scene/gui/reference_rect.h

```diff
@@ -47,6 +48,9 @@ class ReferenceRect : public Control {
 
 	void set_border_color(const Color &color);
 	Color get_border_color() const;
+
+	void set_show_in_game(const bool &enabled);
```

Comment:
Here its preferable to be p_value or p_enabled, while in the external code(visible to user in editor there should be without "p_")

Reply:
Ah, nvm its good as it is